pkg/gofr/datasource/mongo/mongo_test.go

```diff
@@ -79,7 +80,7 @@ func Test_InsertCommands(t *testing.T) {
 		resp, err := cl.InsertOne(context.Background(), mt.Coll.Name(), doc)
 
 		assert.NotNil(t, resp)
-		assert.Nil(t, err)
+		assert.NoError(t, err)
```

Comment:
just curious, were the tests failing everywhere with the latest changes? or is this just changed as part of code quality?